Source/CsDebugScript.Engine/Engine/Utility/DictionaryCache.cs

```diff
@@ -19,10 +20,14 @@ public class DictionaryCache<TKey, TValue>
         /// <summary>
         /// The cached values
         /// </summary>
-        //private Dictionary<TKey, TValue> values = new Dictionary<TKey, TValue>();
         private ConcurrentDictionary<TKey, TValue> values = new ConcurrentDictionary<TKey, TValue>();
 
         /// <summary>
+        /// Indicating whether this collection contains any cached items.
+        /// </summary>
+        private bool isCached;
```

Comment:
True.

---
In reply to: [70602133](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70602133) [](ancestors = 70602133)